.github/workflows/PR_reviews.yml

```
@@ -0,0 +1,32 @@
+name: comment-on-pr example
+on: pull_request
+jobs:
+  review:
+    name: PR review
+    runs-on: ubuntu-latest
+    steps:
+      - name: comment PR
+        uses: unsplash/comment-on-pr@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          msg: "Before a PR is accepted, it must meet the following criteria: \n
+                - [ ] Is the necessary information provided? \n
+                  \t - [ ] Does the PR have a complete description? Does it explain what the PR is attempting to do/fix, does it explain how it does this? \n
+                  \t - [ ] Is there an explanation of why this PR is needed? \n
+                  \t - [ ] Please use the following PR template: https://github.com/tardis-sn/tardis/blob/master/.github/PULL_REQUEST_TEMPLATE.md \n
```

```suggestion
                  \t - [ ] Please use the [TARDIS PR template](https://github.com/tardis-sn/tardis/blob/master/.github/PULL_REQUEST_TEMPLATE.md) \n
```
I hope this doesn't break gh-action ?

---

Seems to work on my own repo!